assert/assertions_test.go

```diff
@@ -605,39 +605,47 @@ func Test_samePointers(t *testing.T) {
 		second interface{}
 	}
 	tests := []struct {
-		name      string
-		args      args
-		assertion BoolAssertionFunc
+		name string
+		args args
+		same BoolAssertionFunc
+		ok   BoolAssertionFunc
```

Comment:
That's perfect. 👌

Can you add a couple more test cases for when the left and right arguments aren't pointers but the other one is, like:
* `args{first: 1, second: p}`
* `args{first: p, second: 1}`

Reply:
Yup, I've added 2 additional test cases now